src/rust/src/feature_writer.rs

```diff
@@ -22,6 +23,7 @@ pub struct FeatureWriter<'a> {
     parts: Vec<flatbuffers::WIPOffset<Geometry<'a>>>,
     properties: Vec<u8>,
     fbb: flatbuffers::FlatBufferBuilder<'a>,
+    pub(crate) bbox: NodeItem,
```

Comment:
hmm what does pub(crate) mean?

Reply:
Makes `bbox` public within the crate only. So it can be accessed in `file_writer.rs`, but it's not visible for users of the library.